### PR TITLE
Update the coding standard

### PR DESCRIPTION
**Pull request by singh264**
Issue: https://github.com/eclipse-openj9/openj9/issues/19261
Co-authored-by: Jason Feng <fengj@ca.ibm.com>
Co-authored-by: Keith W. Campbell <keithc@ca.ibm.com>
Signed-off-by: Amarpreet Singh <Amarpreet.A.Singh@ibm.com>

### CODE REVIEW

**Comment by singh264** [comment]
@tajila @JasonFengJ9 @keithc-ca how can I request your review?

File: doc/CodingStandard.md [hunk 1]
```diff
@@ -421,6 +436,7 @@ if (NULL != myPointer) {
 * The ") {" sequence provides a clear visual indication that the condition has ended and the block has begun.
 * C order of operation rules are complicated; avoid confusion and mistakes by using parentheses to make the order explicit.
 * Combining assignment and tests violates the "Try to do only one thing on each line of code" principle.
+* It seems that when comparing a value with two other values as ((min <= value) && (value <= max)), it is clearer.
```
Comment (keithc-ca) [inline doc/CodingStandard.md:439]: I suggest removing "It seems that".

**Comment by singh264** [comment]
@tajila @JasonFengJ9 @keithc-ca how can I re-request your review?

**Review by keithc-ca — APPROVED** [review]
Clarity comes from the similarity to the mathematical notation $min \le value \le max$ (except that $value$ must be repeated).

**Comment by singh264** [comment]
@tajila @JasonFengJ9 @keithc-ca @babsingh how can I re-request your review?

File: doc/CodingStandard.md [hunk 2]
```diff
@@ -412,6 +412,21 @@ if (NULL != myPointer) {
 }
 ```
 
+* When comparing a value with two other values with <, <=, >, or >= utilize the following format:
```
Comment (babsingh) [inline doc/CodingStandard.md:415]:
```suggestion
* Utilize the below format for comparisons (<, <=, >, or >=) of the form _`min <= value <= max`_ to improve clarity.
```

File: doc/CodingStandard.md [hunk 3]
```diff
@@ -421,6 +436,7 @@ if (NULL != myPointer) {
 * The ") {" sequence provides a clear visual indication that the condition has ended and the block has begun.
 * C order of operation rules are complicated; avoid confusion and mistakes by using parentheses to make the order explicit.
 * Combining assignment and tests violates the "Try to do only one thing on each line of code" principle.
+* When comparing a value with two other values as ((min <= value) && (value <= max)), it is clearer.
```
Comment (babsingh) [inline doc/CodingStandard.md:439]: This can be removed since the above suggestion includes the justification next to the example.

**Comment by singh264** [comment]
@tajila @JasonFengJ9 @keithc-ca @babsingh how can I re-request your review?

File: doc/CodingStandard.md [hunk 4]
```diff
@@ -412,6 +412,17 @@ if (NULL != myPointer) {
 }
 ```
 
+* Utilize the below format for comparisons (<, <=, >, or >=) of the form _min <= value <= max_ to improve clarity.
```
Comment (babsingh) [inline doc/CodingStandard.md:415]: Using brackets and a code block will be better in highlighting the notation than italicizing the text.
```suggestion
* Utilize the below format for comparisons `(<, <=, >, or >=)` of the form `(min <= value <= max)` to improve clarity.
```

**Comment by singh264** [comment]
@tajila @JasonFengJ9 @keithc-ca @babsingh how can I re-request your review?

**Review by babsingh — APPROVED** [review]
Only the coding standards are updated; no more PR builds are needed.

**Pull request merged**